neo/SmartContract/InteropService.cs

```diff
@@ -212,6 +213,17 @@ private static bool Runtime_Serialize(ApplicationEngine engine)
             return true;
         }
 
+        private static bool Runtime_GetInvocationCounter(ApplicationEngine engine)
+        {
+            if (!engine.InvocationCounter.TryGetValue(engine.CurrentScriptHash, out var counter))
+            {
+                counter = 1;
```

Comment:
I don't think this can happen.

Reply:
Yes, I couldn't happen, return false?
```suggestion
                return false;
```